src/graphql/queries/personQuery.js

```diff
@@ -33,6 +33,19 @@ export default
       neighborhood
       region
     }
+    events {
```

Comment:
once again im going to flag that i suspect loading all this at once is going to make loading user profiles too slow... im down to try it though

Reply:
I'm really not following your contention on this one. This particular query is only called once when a Member Profile is visited. The same data is needed for that page whether it's fetched via multiple graphql queries or one. This is not the query on `person` which is used on the member profile page (`community { members }` is used there without any of this additional data being queried).

Reply:
well can you explain to me why loading the Edit Profile page is now quite slow? I assume its because of the many queries happening on the back end that all have to complete before data is returned to the front-end. the way to make a web app feel fast is to load some data immediately and then load more using separate requests

Reply:
I mean performance a big rabbit hole to open up but its a real issue with Hylo right now that I promise is impacting adoption and happiness using it

Reply:
to be clear, I'm not asking for any change right now

Reply:
Thanks for hashing this out a bit more @tibetsprague. In careful look at the Edit Profile page it appears that calling `fetchForCurrentUser` is querying much more than the data needing refresh for the edit profile (i.e. notifications, etc--as you pointed out). The edit page could be made available much more quickly in that case by querying just the fields on `Me` which are being edited instead of using the full `fetchForCurrentUser`, and also potentially taking-out the pending load for `savedSearches` across the whole UserSettings area and pushing it down into a loading indicator on the Edit Saved Searches tab.

* Copying this note onto a new ticket re. the loading time of Edit Profile.